guides/check_definition.schema.json

```diff
@@ -0,0 +1,139 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Trento Check Definition Schema",
+    "additionalProperties": false,
+    "properties": {
+      "id": {
+        "type": "string"
+      },
+      "name": {
+        "type": "string"
+      },
+      "group": {
+        "type": "string"
+      },
+      "description": {
+        "type": "string"
+      },
+      "remediation": {
+        "type": "string"
+      },
+      "facts": {
+        "type": "array",
+        "items": {
+          "$ref": "#/definitions/Fact"
+        }
+      },
+      "values": {
+        "type": "array",
+        "items": {
+          "$ref": "#/definitions/Value"
+        }
+      },
+      "expectations": {
+        "type": "array",
+        "items": {
+          "$ref": "#/definitions/Expectation"
+        }
+      }
+    },
+    "required": [
+      "id",
+      "name",
+      "group",
+      "description",
+      "remediation",
+      "facts",
+      "expectations"
+    ],
+    "definitions": {
+      "Fact": {
+        "title": "Fact",
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "gatherer": {
+            "type": "string"
+          },
+          "argument": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "argument",
```

Comment:
I'm not sure if the `argument` would be always required, but well, let it be by now. It is fine